apps/wallet/tailwind.config.js

```diff
@@ -43,7 +43,7 @@ module.exports = {
                 '4lg': '1rem',
             },
             gridTemplateColumns: {
-                header: '1fr fit-content(160px) 1fr',
```

Comment:
Just FYI - @Jordan-Mysten bumped this to `200px` in https://github.com/MystenLabs/sui/pull/9428/files#diff-c4da9a6700bc507f0e01d8e4b7000a8f7fabcf98ed586b7a75b4e95746c1cd38R46